src/pages/settings/AboutPage/TroubleshootPage.tsx

```diff
@@ -0,0 +1,120 @@
+import React, {useMemo, useState} from 'react';
+import {View} from 'react-native';
+import Onyx from 'react-native-onyx';
+import type {SvgProps} from 'react-native-svg';
+import ConfirmModal from '@components/ConfirmModal';
+import * as Expensicons from '@components/Icon/Expensicons';
+import IllustratedHeaderPageLayout from '@components/IllustratedHeaderPageLayout';
+import LottieAnimations from '@components/LottieAnimations';
+import MenuItemList from '@components/MenuItemList';
+import TestToolMenu from '@components/TestToolMenu';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useEnvironment from '@hooks/useEnvironment';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import * as App from '@userActions/App';
+import * as Report from '@userActions/Report';
+import type {TranslationPaths} from '@src/languages/types';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {OnyxKey} from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import SCREENS from '@src/SCREENS';
+
+const keysToPreserve: OnyxKey[] = [
+    ONYXKEYS.ACCOUNT,
```

Comment:
@TMisiukiewicz Hey, just wondering if there was any specific reason not to include `ONYXKEYS.USER` here? We're considering adding it in https://github.com/Expensify/App/issues/49087, but wanted to check first if there are any good arguments against doing so.

Reply:
@jjcoffee some time passed so I don't remember the exact context, but I think adding `ONYXKEYS.USER` should be fine here